public/server.py

```diff
@@ -0,0 +1,13 @@
+from http.server import HTTPServer, SimpleHTTPRequestHandler
```

Comment:
Please write this server in node(see sample in torus-website) and keep it outside of src folder and not in public folder
Reason for writing in node: This script is only for python 3 and has problems for people using different versions.

Reply:
Resaon for server.py -> I added the python server file for local development (So that the extension can inject the locally developed script). 


New server.js created.